Mathlib/Data/List/Basic.lean

```diff
@@ -275,7 +275,7 @@ theorem forall_mem_of_forall_mem_cons {p : α → Prop} {a : α} {l : List α} (
 
 #align list.forall_mem_append List.forall_mem_append
 
-theorem not_exists_mem_nil (p : α → Prop) : ¬∃ x ∈ @nil α, p x :=
+theorem not_exists_nil (p : α → Prop) : ¬∃ x ∈ @nil α, p x :=
```

Comment:
```suggestion
theorem not_exists_mem_nil (p : α → Prop) : ¬∃ x ∈ @nil α, p x :=
```

Reply:
This lemma already exists (line 75 of that file): as-is, this change doesn't compile

Reply:
Can you deduplicate then?

Reply:
AFAICT, this theorem is never used in mathlib: I've removed it (and kept the align, as it points to the other lemma).